src/components/buttons/Button/Button.ios.js

```diff
@@ -34,93 +33,68 @@ const Container = styled(Centered)`
   ${({ showShadow }) => (showShadow ? shadowStyles : '')}
   ${({ size }) => padding(...ButtonSizeTypes[size].padding)}
   background-color: ${({ backgroundColor }) => backgroundColor};
-  border-radius: ${({ type }) => (type === 'rounded' ? 14 : 50)};
+  border-radius: ${({ borderRadius }) => borderRadius};
   flex-grow: 0;
 `;
 
 const shouldRenderChildrenAsText = children =>
   isArray(children) ? isString(children[0]) : isString(children);
 
-const Button = ({
-  backgroundColor,
+export default function Button({
+  backgroundColor = colors.grey,
   borderColor,
   borderOpacity,
   borderWidth,
   children,
-  color,
+  color = colors.white,
   containerStyles,
   disabled,
   onPress,
-  showShadow,
-  size,
+  showShadow = true,
+  size = 'default',
   style,
   textProps,
-  type,
+  type = ButtonShapeTypes.pill,
   ...props
-}) => (
-  <ButtonPressAnimation
-    {...pick(props, Object.keys(ButtonPressAnimation.propTypes))}
-    backgroundColor={backgroundColor}
-    disabled={disabled}
-    onPress={onPress}
-    style={[props.style, { borderRadius: type === 'rounded' ? 14 : 50 }]}
```

Comment:
👀️ notice how previously it was trying to access `props.style` even though we already deconstructed the `style` key out of the `prop` object on line 56^^ 

^^^ fixing this is the most impactful change ive made in this file. everything else is just updating the syntax patterns to match our latest patterns.